hutch_python/tests/test_happi.py

```diff
@@ -23,7 +23,7 @@ def test_happi_objs():
     # Only select active objects
     lc = get_lightpath(db, 'tst')
     objs = get_happi_objs(db, lc, 'tst')
-    assert len(objs) == 2
+    assert len(objs) == 4
```

Comment:
It looks to me like we only have 3 devices on tst- where does the 4th come from? Maybe I misunderstand, I'll re-read the function text.
EDIT: I see this matches the "STANDARD" load below, which means it's consistent.
Does this mean that "STANDARD" doesn't take into account the branching device 5 at the end?